packages/ckeditor5-table/tests/manual/tableproperties.html

```diff
@@ -23,6 +23,33 @@
 							<td style="background:transparent;"><p>B2</p></td>
 							<td><p>C3</p></td>
 						</tr>
+						<tr>
+							<td colspan="3" style="border: 1px solid brown;">
+								<table style="border: 1px solid green;background:#fdfd77;width:200px;height:140px">
+									<thead>
+										<tr>
+											<td><p>Column X</p></td>
+											<td style="border:2px dashed black;background-color: lightblue"><p>Column Y</p></td>
+											<td style="background: palevioletred"><p>Column Z</p></td>
+										</tr>
+									</thead>
+									<tbody>
+										<tr>
+											<td style="border-color: deeppink deepskyblue;border-style: solid;border-width: 1px;border-bottom-width: 2px;border-right-style: dotted;">
+												<p>X1</p>
+											</td>
+											<td style="border: 1px solid blue;"><p>Y1</p></td>
```

Comment:
Spaces or no after `:`? IDK if you copied this piece from somewhere else, but let's try to be consistent (in L45-47 you have no space). Let's use the same format that the editor uses when outputting content.

Reply:
Hmmm, OTOH, a part of table props' logic is how it parses inline styles, so that may be the reason for lack of consequence. So I take the above back – let's keep it this way.